SCAN.md

```diff
@@ -1,5 +1,23 @@
 Documentation on the Seattle Coronavirus Assessment Network (SCAN) project
 
+- [Adding a new study arm](#adding-a-new-study-arm)
+- [Adding a new language](#adding-a-new-language)
+
+## Adding a new study arm
+
+1. First, answer the questions listed in the [barcodes documentation] pertaining to creating new collections.
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
+5. Update the REDCap DET ETL with the new study's project ID and purview (see [example]).
+6. Add a new cronjob for the new REDCap DET ETL.
+7. Decide if we need to manually generate DETs (for e.g. if the new study arm uses the REDCap mobile app or API for enrollments)
+   > Note: even if the mobile app or API are being used for enrollments in this new study, as long as a field like `back_end_scan_date` is updated manually during QC, then we'll receive DETs as usual for this project.
```

Comment:
> we'll receive DETs as usual for this project.

Not _quite_ DETs as usual, but we'll get at least one DET for the instrument containing the manually-updated field. If we have other required instruments, that DET must come _after_ those instruments are complete. This works fairly well if the manually-updated field is in a later instrument, like kit unboxing.

Reply:
I addressed this in the latest commit.